huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -1085,16 +1126,16 @@ func resourceComputeInstanceV2Delete(d *schema.ResourceData, meta interface{}) e
 			logp.Printf("[DEBUG] Waiting for instance (%s) to stop", d.Id())
 			pending := []string{"ACTIVE"}
 			target := []string{"SHUTOFF"}
-			timeout := d.Timeout(schema.TimeoutDelete)
-			if err := waitForServerTargetState(ecsClient, d.Id(), pending, target, timeout); err != nil {
-				return fmtp.Errorf("State waiting timeout: %s", err)
+			timeout := d.Timeout(schema.TimeoutCreate)
+			if err := waitForServerTargetState(ctx, computeClient, d.Id(), pending, target, timeout); err != nil {
```

Comment:
it seems that we should not change the timeout and client

Reply:
fixed